docs/authorization.md

```diff
@@ -0,0 +1,121 @@
+# Authorization
+
+Stitch provides ability manage access to data using authorization policies. These policies are attached to schema fields that require authorized access.
+
+## Policy definition
+
+Policy is Stitch resource like schema. It can be created, altered or removed using Registry API.
+
+The policy object has the following properties:
+
+| property | description                                                                                                                                     | mandatory |
+| -------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | --------- |
+| metadata | Contains two fields name and namespace.</br> Namespace is a logical group of policies.</br> Namespace and name pair is policy unique identifier | true      |
+| type     | Policy type.</br> Currently only [OPA](https://www.openpolicyagent.org) policies are supported                                                  | true      |
+| code     | Policy body.</br>Currently only [Rego](https://www.openpolicyagent.org/docs/latest/policy-language) language is supported.</br>                 | true      |
+| args     | Argument definitions.</br> These arguments is available in the policy body.                                                                     | false     |
+| query    | Graphql query to provide additional data to policy.</br> Its result is available in the policy body.                                            | false     |
+
+Example of policy definition:
```

Comment:
I think this example will be overwhelming for anyone reading the docs that is unfamiliar with the feature.

A few notes about examples:

- I prefer the approach taken in the spec of starting with very simple examples using only the most basic of features and then using more examples to each introduce and showcase an additional feature. IMO these examples would be the most important part of the documentation for a feature like this.

- Examples are clearer when they use some obvious context for the namespace and name, like users, movies, accounts etc. (the context doesn't need to actually be defined anywhere).
For example, instead of `my_namespace` and `policy1` you can use the namespace `users` and policy name 'onlyAdmins'.

- I think we should have an example that clearly showcases the use of JWT in the policy, as that can potentially be a common use case